docs/extensions/services/working-with-users.md

```diff
@@ -0,0 +1,104 @@
+---
+description: Learn about the required services when working with users and granting them access control.
+contributors: Esther Agbaje
+---
+
+# Working with Users
+
+Directus provides a comprehensive set of services that facilitate seamless collaboration with users while enforcing
+access control to specific roles. They include:
+
+- `UsersService`: Extends the `ItemsService` and is used to manage user accounts and perform CRUD operations on user
+  profiles
+- `RolesServices`: Extends the `ItemsService` and enables you to assign roles to users.
+- `PermissionsServices`: Extends the `ItemsService` and allows you to manage the permissions associated with each role.
```

Comment:
Remove 'Extends the ItemsService' at the start and get to what they do. Then just add it as a sentence under the list. 'All three of these services extend the Items service so are used in a very similar way' or something like that linking to the ItemsService page.